sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java

```diff
@@ -696,7 +818,6 @@ static void validateMultiPageDataLabeled(List<RecognizedForm> actualRecognizedFo
                 assertNotNull(formField.getName());
                 assertNotNull(formField.getValue());
                 assertNotNull(formField.getValueData().getText());
```

Comment:
Why was this removed?

Reply:
my bad. It used to cause some tests failed with previous implementation, but no longer a case now. Will bring it back.